### ISSUE TITLE
Cannot reproduce the reported results on wisdm

### ISSUE THREAD

**Issue by conditionWang** [mqwfrog/MHCCL#8]
Hello, I cannot reproduce good results on wisdm with the following parameter settings, could you share the exact parameter setting for wisdm?

```
python main.py \
--dataset_name wisdm \
--lr 0.03 \
--batch_size 128 \
--mlp --cos \
--layers 3 \
--posi 2 \
--negi 100 \
--posp 3 \
--negp 4 \
--gpu 7 --epochs 150
--mask_layer0 \
--mask_mode 'mask_threshold'
```

**Comment by zlyzmrr** [comment]
Hello, I'm also facing the same issue as you. Did you achieve good performance by making further modifications to new parameters?

**Comment by conditionWang** [comment]
> Hello, I'm also facing the same issue as you. Did you achieve good performance by making further modifications to new parameters?

Nope. I cannot reproduce effective results with a number of tries.
